pkg/flow/logging/logger.go

```diff
@@ -60,6 +67,35 @@ func New(w io.Writer, o Options) (*Logger, error) {
 	if err := l.Update(o); err != nil {
 		return nil, err
 	}
+
+	return l, nil
+}
+
+// NewDeferred creates a new logger with the default log level and format.
+// The logger is not updated during initialization.
+func NewDeferred(w io.Writer, o Options) (*Logger, error) {
```

Comment:
"o Options" is not used in this function and can be removed

Reply:
Removed :D